arch/cpu/cc26xx-cc13xx/rf-core/prop-mode.c

```diff
@@ -795,6 +796,12 @@ read_frame(void *buf, unsigned short buf_len)
     entry->status = DATA_ENTRY_STATUS_PENDING;
   }
 
+  if(rx_is_full) {
+    rx_is_full = false;
```

Comment:
The comparison and setting to false should be inside a critical section, since the interrupt may fire after the comparison, but before setting the value to false. In this case, we may end up with the variable being set to false erroneously. Please use the functions in critical.h